src/Synapse/Synapse/Commands/ManagementCommands/Workspace/UpdateAzureSynapseWorkspace.cs

```diff
@@ -116,6 +120,7 @@ public override void ExecuteCmdlet()
                     }
                 }
             } : null;
+            patchInfo.WorkspaceRepositoryConfiguration = this.IsParameterBound(c => c.GitRepository) ? this.GitRepository.ToSdkObject() : null;
```

Comment:
We have an issue here including but not limited to `WorkspaceRepositoryConfiguration`: we cannot use this PowerShell cmdlet to disable features. This is we are using PATCH API. When we set the value to `null`, then the resource will stay unchanged. I am wondering what the payload look like when you disable the feature through Synapse Studio? In addition, I suppose this is a command scenario, is there any good examples that we can follow of which cmdlets that consume PATCH API?